src/widgets/NavigationBar/ui/Notifications.tsx

```
@@ -65,73 +67,116 @@ export default function Notifications() {
-        alt="notification-icon"
-        priority
-        className="cursor-pointer"
-        onClick={() => setOpen(!open)}
-      />
-      {open && (
-        <div className="absolute left-0 top-full mt-2 w-80 bg-card border border-border rounded-[10px] shadow-hover z-999 bg-[#0c151c]">
-          <div className="p-4 border-b border-border">
-            <h3 className="font-medium text-foreground">알림</h3>
+      {/* 알림 아이콘 with 배지 */}
+      <div className="relative cursor-pointer" onClick={() => setOpen(!open)}>
+        <Image
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)